public/widget.js

```diff
@@ -80,7 +80,7 @@ var CONTENT = {
         'cy-gb': 'Blwyddyn dramor yn ddewisol'
     },
     'dataFor': {
-        'en-gb': 'Data for: ',
+        'en-gb': 'Data for ',
```

Comment:
Did you intend to remove the `:` as it's still on the welsh

Reply:
Yes, will change welsh also.